README.md

```diff
@@ -155,7 +155,7 @@ class ZoneMaterializer
 
   persist_to :zone
 
-  source_key :source_id do |url|
+  source_key :source_id do |url, captured_attributes|
```

Comment:
not sure if we should pass in the response body or the captured attributes

Reply:
Hum... I think it'd make more sense to pass the body in this case or whatever `capture` is receiving when we use the block form.

Reply:
Yep, I think that would allow us to access attributes that we don't want to capture